internal/devbox/providers/nixcache/setup.go

```diff
@@ -88,7 +89,9 @@ func (s *setupTask) Run(ctx context.Context) error {
 	}
 
 	err = nix.IncludeDevboxConfig(ctx, s.username)
-	if err != nil {
+	if errors.Is(err, nix.ErrUnknownServiceManager) {
+		ux.Fwarning(os.Stderr, "Devbox configured Nix to use your organization's private cache. Please restart the Nix daemon and re-run Devbox.\n")
+	} else if err != nil {
```

Comment:
Would this show up in CICD even if they don't need to do it?

Reply:
Fixed.